specs/2019-04/Windows-Plan.md

```diff
@@ -0,0 +1,98 @@
+# App Center Windows Application Proposal 
+
+This plan details the proposed improvements for App Center’s Windows application offering along with a comparison of App Center and HockeyApp. This is work in progress and will be iterated upon based on any internal updates and feedback from the community. We expect to start work on the planned items in the second quarter of 2019. 
+
+
+## App Center’s Current Offering 
+### Diagnostics
+
+- Crash reporting for UWP (.NET native) apps distributed through the Windows store 
+> Note: there are some known limitations and missing features for UWP apps. Find more details in our [Windows documentation](https://docs.microsoft.com/en-us/appcenter/diagnostics/windows-support). 
+
+### Distribution
+
+- Distributing UWP and Unity UWP apps using the APPX or MSIX package format
+- Unofficial support to distribute WPF and WinForms apps when packaged as APPX or MSIX using the [Windows Application Packaging](https://docs.microsoft.com/en-us/windows/uwp/porting/desktop-to-uwp-packaging-dot-net)
+
+### Analytics 
+
+- Analytics SDK for UWP apps and all of its functionalities listed in App Center's [analytics documentation](https://docs.microsoft.com/en-us/appcenter/analytics/).  
+
+### Build
+- Build support for UWP apps
+
+## Planned Improvements 
+
+### Diagnostics 
+
+- Crash reporting capabilities for store and sideloaded UWP (.NET Native) and WPF (.NET Framework) apps aligned with other platforms supported in App Center. Find more details on App Center Diagnostics features in our [documentation](https://docs.microsoft.com/en-us/appcenter/diagnostics/features).  
+
+> Note: this does not include handled exceptions and [custom properties](https://docs.microsoft.com/en-us/appcenter/diagnostics/features#key-value-pairs).  
+ 
+
+### Distribution 
+
+- Official support for WPF applications 
```

Comment:
For WPF, This really needs to also include zipped EXE/MSI installers. 
Reasons for this : 
1. Hockeyapp supported it
2. ISVs that create WPF apps still have a significant amount of Win7 customers, beta testers, and internal QA/Test infrastructure( to support Win7 ) where MSIX isn't practical and APPX isn't possible. 
For WPF distribution
- From our standpoint, there isn't any expectation of delivery to a store or in-app updates.   I would think most ISVs have had to build or adopt their own in-app update mechanism for Win7 anyway.
- Just being able to download the zipped EXE/MSI from a webpage within AppCenter ( like it current works and it worked under HockeyApp ) would be fine.
- It would be nice to reduce the clicks to get the download to as few as possible.

Reply:
MSIX will also support [Win7](https://github.com/Microsoft/msix-packaging/tree/master/preview/Win7Msix). Any reason we can't support both? If we support UWP, I would expect the deployment model for any other MSIX package (whether it is WinForms or WPF) to be able to share the same App Center deployment infrastructure.